src/api/java/mezz/jei/api/recipe/category/IRecipeCategory.java

```diff
@@ -36,9 +37,22 @@
 	/**
 	 * Returns the localized name for this recipe type.
 	 * Drawn at the top of the recipe GUI pages for this category.
+	 * @deprecated Prefer calling and using {@link #getTitleAsTextComponent()}.
```

Comment:
I like putting a version on deprecation warnings so people know how old it is
```suggestion
	 * @deprecated since JEI 7.6.4, prefer calling and using {@link #getTitleAsTextComponent()}.
```